ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -333,8 +333,9 @@ export default defineComponent({
           !!location.landDistrict || !!location.plan || !!location.exceptionPlan || !!location.reserveNumber
       }),
       landOwnershipLabel: computed(() => {
+        if (getMhrRegistrationOwnLand.value === null) return '(Not Entered)'
```

Comment:
`null` is actually pretty explicit, might also want to check for `undefined` or an empty string `''`. 
All boils down to how the property was initialized. 
Either way, `!getMhrRegistrationOwnLand.value` might work for your condition, as all the above are falsy.

Reply:
ah i see, sort of a weird scenario when dealing with a boolean, does something display when it's false?
Sorry maybe my suggestion was confusing then. If the value is initialized as `null` than perhaps your first iteration was just fine, apologies!
I was thinking `if (!getMhrRegistrationOwnLand.value) return '(Not Entered)'` but that might not work if you need to display something when it's false.
So yea, 100% your call, null would suffice too if thats how its initialized.